articles/metadata/apiv2.md

```diff
@@ -108,9 +113,11 @@ The user's profile now appears as follows:
 }
 ```
 
-**Note:** Only properties at the root level are merged in to the object. All others replace the existing value. The following example demonstrates an example of this merging.
+### Merging 
+
+Only properties at the root level are merged into the object. All lower-level properties will be replaced.
 
-If we were to add the user's work address as an additional inner property, we have to send over the entire address block. Because the `addresses` object is a root-level property, it will be merged into the final JSON object representing the user, but its *properties* will not.
+For example, to add a user's work address as an additional inner property, you would have to include the complete contents of address property. Since the `addresses` object is a root-level property, it will be merged into the final JSON object representing the user, but its sub-properties will not.
```

Comment:
"..the complete contents of the `address` property" reads better than "..the complete contents of address property" in my opinion. Just a suggestion.